From c6d73a291ad1ca067a63838e1f2d8a914ae42452 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Tue, 21 Jul 2020 08:58:32 +0800
Subject: [PATCH 1/1] Test case for .inc file

Description: Change the Location value of the 'ApProcedureLocation' in the UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc b/UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc
index 58ef369342a7..e19683ad4bb2 100644
--- a/UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc
+++ b/UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc
@@ -22,7 +22,7 @@ CPU_SWITCH_STATE_LOADED       equ        2
 LockLocation                  equ        (RendezvousFunnelProcEnd - RendezvousFunnelProcStart)
 StackStartAddressLocation     equ        LockLocation + 08h
 StackSizeLocation             equ        LockLocation + 10h
-ApProcedureLocation           equ        LockLocation + 18h
+ApProcedureLocation           equ        LockLocation + 1Ah
 GdtrLocation                  equ        LockLocation + 20h
 IdtrLocation                  equ        LockLocation + 2Ah
 BufferStartLocation           equ        LockLocation + 34h
-- 
2.27.0.windows.1

